test/spec/grammar/ast_builder_test.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable key-spacing, no-unused-expressions */
+const expect = require('chai').expect;
+const parse = require('../../../lib/dsl/poc/api').parse;
+const buildAst = require('../../../lib/dsl/poc/ast_builder').buildAst;
+
+describe('ASTBuilder', () => {
+  it('can build an ast for a simple entity', () => {
+    const input = 'entity Person { name string }';
+    const parseResult = parse(input);
+    expect(parseResult.lexErrors).to.be.empty;
+    expect(parseResult.parseErrors).to.be.empty;
+
+    const ast = buildAst(parseResult.cst);
+    expect(ast).to.deep.equal({
+      constants: [],
+      entities:  [
+        {
+          fields:    [
+            {
+              name:        'name',
+              type:        'string',
+              validations: []
+            }
+          ],
+          name:      'Person',
+          tableName: undefined
+        }
+      ]
+    });
+  });
+
+  it('can build an ast for a complex JDL', () => {
+    const input = `
+    entity Person (HRXD1)
```

Comment:
Rather:
```
entity XXX {
   ...
}
```

Reply:
I don't understand the comment

Reply:
The syntax
```
entity
{
   ...
}
```
should be avoided, the left curly braces should be on the same line as the `entity` (looks like Java).